src/sentry/testutils/fixtures.py

```diff
@@ -751,6 +751,9 @@ def create_sentry_app(self, name=None, organization=None, published=False, scope
             'scopes': scopes,
             'webhook_url': webhook_url,
             'events': [],
+            'schema': json.dumps({
```

Comment:
Since `schema` isn't required, it seems weird to me to have it in as a default - also we don't do that for `events`.